### PR TITLE
Documentation translated into Spanish

### PR DESCRIPTION
**Pull request by MoonKIE666**
I have translated all the documents included in the \ doc \ directory into Spanish, and I have given them the acronym "es-" at the beginning of the name of each file

### CODE REVIEW

File: doc/es-Development.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Asegúrese de comprender el flujo de trabajo de GitHub: https://guides.github.com/introduction/flow/index.html
+
+Mantenga las solicitudes de extracción enfocadas solo en una cosa, ya que esto hace que sea más fácil combinar y probar de manera oportuna.
+
+Si necesita ayuda con las solicitudes de extracción, hay guías en GitHub aquí:
+
+https://help.github.com/articles/creating-a-pull-request
+
+The main flow for a contributing is as follows:
```
Comment (HazardCreative) [inline doc/es-Development.md]: Check this line.

File: doc/es-Cluster.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+﻿# Cluster
```
Comment (HazardCreative) [inline doc/es-Cluster.md]: No need to include files that have not changed.

File: doc/es-Hardware Setup.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+### Soporte LED WS2812b
+Los pines en el cuadro verde son los que ya están en uso en el temporizador. Los pines en el cuadro rojo es donde se debe conectar la señal y la tierra de los LED ws2812b. Los LED requerirán una fuente de alimentación separada. Consulte el soporte de LED WS2812b en Configuración de software.
+![led wiring](img/GPIO.jpg)
+
+### Sensores adicionales
+Se pueden conectar sensores (como BME280 e INA219) al bus I2C y a los pines de alimentación. Consulte los archivos '..._ sensor.py' en el directorio "src / interface" para ver ejemplos de implementación. Los sensores deben especificarse en el archivo "src / server / config.json": en la siguiente configuración de muestra, un sensor BME280 está configurado en la dirección I2C 0x76 (como "Clima") y los sensores INA219 están configurados en 0x40 y 0x41 .
+```
+    "SENSORS": {
+            "i2c:0x76": {
+                    "name": "Climate"
```
Comment (HazardCreative) [inline doc/es-Hardware Setup.md]: Do you want to translate these labels?

File: doc/es-Software Setup.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+Automatic node configuration is also possible by grounding of hardware pins. Set NODE_NUMBER to 0, then tie these pins to ground:
+
+node #1: ground pin D5<br/>
+node #2: ground pin D6<br/>
+node #3: ground pin D7<br/>
+node #4: ground pin D8<br/>
+node #5: ground pin D5 and pin D4<br/>
+node #6: ground pin D6 and pin D4<br/>
+node #7: ground pin D7 and pin D4<br/>
+node #8: ground pin D8 and pin D4
```
Comment (HazardCreative) [inline doc/es-Software Setup.md]: Could translate these as well.

File: doc/es-Software Setup.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+Si no hay configurados nodos de hardware, el servidor funcionará utilizando nodos simulados (simulados). En este modo, se puede explorar y probar la interfaz web-GUI.
+
+Para ver la interfaz web-GUI, abra un navegador web e ingrese en la barra de direcciones: ```localhost:5000``` (Si el valor HTTP_PORT en la configuración ha cambiado, use ese valor en lugar de 5000). Si el servidor se está ejecutando, debería aparecer la página principal de RotorHazard. Tenga en cuenta que las páginas reservadas para el director de carrera (Administrador / Configuración) están protegidas con contraseña con el nombre de usuario y la contraseña especificados en la configuración.
+
+<br/>
+
+-----------------------------
+
+See Also:<br/>
```
Comment (HazardCreative) [inline doc/es-Software Setup.md]: Translate this line?

File: doc/es-Cluster.md (hunk 2 above)
Comment (MoonKIE666) [reply doc/es-Cluster.md]: I don't know what happened here, I had translated this

File: doc/es-Development.md (hunk 1 above)
Comment (MoonKIE666) [reply doc/es-Development.md]: Do it

File: doc/es-Cluster.md (hunk 2 above)
Comment (MoonKIE666) [reply doc/es-Cluster.md]: Do it

File: doc/es-Hardware Setup.md (hunk 3 above)
Comment (MoonKIE666) [reply doc/es-Hardware Setup.md]: Do it

File: doc/es-Software Setup.md (hunk 4 above)
Comment (MoonKIE666) [reply doc/es-Software Setup.md]: Do it

File: doc/es-Software Setup.md (hunk 5 above)
Comment (MoonKIE666) [reply doc/es-Software Setup.md]: Do it

**Pull request merged**